src/server/app.js

```diff
@@ -46,4 +42,9 @@ app.use((err, req, res) => {
 	else res.send(`<h1>${err.status} Server Error</h1>`);
 });
 
+// This is after the other app.XXX entries as having it before seems to cause issues. See OED issue #86.
```

Comment:
Minor thing, but this comment isn't super helpful or descriptive. I would recommend removing it or explaining the issue more.